docs/bridge_service.md

```diff
@@ -97,7 +97,66 @@ sequenceDiagram
 This paragraph explains a set of endpoints, that are necessary in order to conduct the bridge flow described above.
 
 ### Get bridges
-<!-- TODO: @temaniarpit27 -->
+Retrieves the bridge(s) for a specified network, supporting pagination. The bridges represent the `BridgeEvent` events emitted by the bridge contract.
+
+#### Parameters
+
+| **Name**           | **Type**    | **Description**                                             | **Required** | **Notes**                       |
+|--------------------|-------------|-------------------------------------------------------------|--------------|---------------------------------|
+| `networkID`        | `uint32`    | ID of the network to fetch token mappings from.             | Yes          | `0` for L1(otherwise L2)        |
```

Comment:
```suggestion
| `networkID`        | `uint32`    | ID of the network to fetch bridges from.             | Yes          | `0` for L1 (otherwise L2)        |
```